Atoms/HitDetection/HurtBox.cs

```diff
@@ -0,0 +1,33 @@
+using Godot;
+using System;
+
+public class HurtBox : Area2D
```

Comment:
how is a hitbox different from a hurtbox?

Reply:
Oops I actually got them twisted. Hurt boxes are often attached to weapons/attacks/etc that *deal* damage whereas Hitboxes are attached to the player/enemy/object that *receives* damage